### PR TITLE
Spyral Fix

### PR DESCRIPTION
**Pull request by KServantes**
Updated SPYRAL Sleeper text in database to "Must be Special Summoned."
Fixed SPYRAL Sleeper and SPRYAL GEAR - Last Resort scripts.

### CODE REVIEW

File: script/c100912086.lua [hunk 1]
```diff
@@ -1,95 +1,101 @@
...
-		Duel.Hint(HINT_SELECTMSG,tp,HINTMSG_SPSUMMON)
-		local g=Duel.SelectMatchingCard(tp,c100912086.spfilter,tp,LOCATION_HAND+LOCATION_DECK+LOCATION_GRAVE,0,1,1,nil,e,tp)
-		if g:GetCount()>0 then
-			Duel.SpecialSummon(g,0,tp,tp,false,false,POS_FACEUP)
-		end
-	end
+--SPYRAL Sleeper
+--Scripted by Eerie Code
+function c100912086.initial_effect(c)
+	c:EnableUnsummonable()
```
Comment (TheOnePharaoh) [inline script/c100912086.lua:4]: You shouldn't remove the revive limit

Comment (TheOnePharaoh) [inline script/c100912086.lua]: also it's best to leave the changes in the beta cards to Eerie

**Review by TheOnePharaoh — CHANGES_REQUESTED** [review]

**Comment by TheOnePharaoh** [comment]
also it's best to leave the changes in the beta cards to Eerie

File: script/c100912086.lua (hunk 1 above)
Comment (KServantes) [reply script/c100912086.lua:4]: It's not a Semi-Nomi.

**Comment by EerieCode** [comment]
Honestly, @KServantes , my biggest issue here is that, with whatever you did to the formatting, it's hard to see what exactly has been changed. If you could change the formatting so that we see only the things you've changed, that'd definitely make things easier. That said, I do have a few things to say:

- Why _c:EnableUnsummonable()_ in Sleeper? It's a Nomi monster, kinda like "Dystopia the Despondant": that's the whole reason for the text change. c:EnableReviveLimit() should suffice.
- Also in Sleeper, during the selection in the last effect, use this line instead:

_local g=Duel.SelectMatchingCard(tp,aux.NecroValleyFilter(c100912086.spfilter),tp,LOCATION_HAND+LOCATION_DECK+LOCATION_GRAVE,0,1,1,nil,e,tp)_

It will check for Necrovalley.
- In Last Resort, at the end of the .dircost function, you should put _Duel.SendtoGrave(g,REASON_COST)_ to fix the cost.

I'll tell you if I find anything else after you upload the reformatted files: if you can't do that, I can upload these fixes along with the new cards in about 8 hours: in which case, it'd be better to close this branch to avoid a huge conflict.

**Comment by KServantes** [comment]
@EerieCode No idea why the formatting came out that way really but I'll reopen a new pull later.

**Comment by TheOnePharaoh** [comment]
it would be best if Eerie did it because he can check any fixes made in fluo as well

**Comment by EerieCode** [comment]
https://github.com/Ygoproco/Live2017/pull/197

He already made the new pull, I've already checked it and it should be fine

**Pull request closed without merging**